web/static/httprest/connect/success.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
Let's write a success template and use it everywhere. Currently there are 9 different files.

Reply:
I agree, but will do it in a separate horizontal PR.

Reply:
https://linear.app/autokitteh/issue/ENG-677/improvements-in-integration-web-uis